plone/app/upgrade/v60/profiles/to6004/registry.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<registry>
+  <record name="plone.resources.development">
+    <field type="plone.registry.field.Bool">
+      <title>Frontend development mode</title>
+      <required>False</required>
+    </field>
+  </record>
+
+  <!-- Load default bundles with default values for all fields (e.g. to get rid of a depdendency on plone.base) -->
+  <records prefix="plone.bundles/bootstrap-js" interface='Products.CMFPlone.interfaces.IBundleRegistry' purge="true">
+    <value key="enabled">False</value>
+    <value key="jscompilation">++plone++static/bundle-bootstrap/js/bootstrap.bundle.min.js</value>
+    <value key="csscompilation"></value>
+    <value key="expression"></value>
+    <value key="depends"></value>
+    <value key="load_async">False</value>
+    <value key="load_defer">False</value>
+  </records>
+
+  <records prefix="plone.bundles/plone" interface='Products.CMFPlone.interfaces.IBundleRegistry' purge="true">
+    <value key="enabled">True</value>
+    <value key="jscompilation">++plone++static/bundle-plone/bundle.min.js</value>
+    <value key="csscompilation"></value>
+    <value key="expression"></value>
+    <value key="depends"></value>
+    <value key="load_async">False</value>
+    <value key="load_defer">False</value>
+  </records>
+
+  <records prefix="plone.bundles/plone-logged-in" interface='Products.CMFPlone.interfaces.IBundleRegistry' purge="true">
+    <value key="merge_with">logged-in</value><!-- needs to be here -->
+    <value key="enabled">False</value>
+    <value key="expression">python: False</value>
+    <value key="jscompilation">++plone++static/plone-logged-in-compiled.min.js</value><!-- needs to be here -->
+    <value key="csscompilation"></value>
+    <value key="depends"></value>
+    <value key="load_async">False</value>
+    <value key="load_defer">False</value>
+  </records>
+
```

Comment:
I am missing the `jquery` and `plone-legacy` bundles here. In a new site they are available, but not in a migrated 5.2 site.
Because jQuery is missing, a lot of javascript does not work, including the Resource Registries themselves.